### PR TITLE
Casing changes around the app

### PR DESCRIPTION
**Pull request by alastairjardine**
Few string changes – all lowercase to Sentence case to be consistent with other Guardian tools

### CODE REVIEW

**Comment by akash1810** [comment]
:+1:

**Comment by theefer** [comment]
Screenshot please?

**Comment by kenoir** [comment]
Might be a good idea for someone to pick this up and fill in the blanks for other places in the app where the casing is inconsistent.

I can pick this up after i'm done with :cloud: :scroll:

**Comment by alastairjardine** [comment]
@theefer Will do – might need someone to put this on an environment for me though…any takers?

**Comment by theefer** [comment]
Is it ready to be put on TEST?

**Comment by alastairjardine** [comment]
@theefer Yeah, let's have a look. Would make it obvious where I've missed at the very least.

**Comment by akash1810** [comment]
Should tooltips be sentence cased too?

**Comment by alastairjardine** [comment]
@akash1810 Aye. Must've missed them.

File: kahuna/public/js/components/gr-panel/gr-panel.html [hunk 1]
```diff
@@ -273,7 +273,7 @@
                 <div class="metadata-line">
                     <gr-icon>star_half</gr-icon>
                     <span ng:if="!ctrl.archiving">{{ctrl.archivedCount}} archived</span>
-                    <span ng:if="ctrl.archiving">saving...</span>
+                    <span ng:if="ctrl.archiving">Saving...</span>
```
Comment (jamesgorrie) [inline kahuna/public/js/components/gr-panel/gr-panel.html]: Could we add `…` while we're here too pretty please?

Comment (alastairjardine) [inline kahuna/public/js/components/gr-panel/gr-panel.html]: Absolutely

**Comment by alastairjardine** [comment]
Help!
Struggling to find:
screen shot 2015-08-13 at 14 51 58
screen shot 2015-08-13 at 14 51 55

**Comment by akash1810** [comment]
https://github.com/guardian/grid/blob/master/kahuna/public/js/upload/file-uploader.html#L13
https://github.com/guardian/grid/blob/master/kahuna/public/js/search/query.html#L24

**Comment by alastairjardine** [comment]
@akash1810 Ta. Done.

**Comment by theefer** [comment]
Couple of remaining misses

in the gr-panel:

image


image preview:

image
(this one is generated programmatically so probably easier if one of us picks it up, it's not the end of the world for an initial version though)


Personally I find the casing more shouty and distracting, but I don't mind that much, so :+1: once the gr-panel is amended.

**Comment by alastairjardine** [comment]
Hmm – looking at test myself and the By and Labels are both correct.

**Comment by theefer** [comment]
Are you sure you're looking at the gr-panel (on the search results page side panel), not the individual image preview page?

**Comment by alastairjardine** [comment]
@theefer Doh. Can you give me a file hint?

**Comment by theefer** [comment]
https://github.com/guardian/grid/blob/master/kahuna/public/js/components/gr-panel/gr-panel.html#L178
https://github.com/guardian/grid/blob/master/kahuna/public/js/components/gr-panel/gr-panel.html#L245

I think

**Comment by alastairjardine** [comment]
Think this is now 'done'.

(so the two panels are different? Any idea if we'll make them refer to the same source? I'd assumed it was always using the same source)

**Comment by theefer** [comment]
> Any idea if we'll make them refer to the same source?

Hopefully soon, that's part of some tech debt unfortunately.

:+1:

**Pull request merged**